### ISSUE TITLE
WIndows 8.1

### ISSUE THREAD

**Issue by ChainlessMind** [norecha/pokubot#13]
I have read that others have gotten Windows 8.1 to work with pokubot but I have failed now for 24 hours. Is it possible on 8.1 currently or will there be a version that is in the future?

**Comment by norecha** [comment]
What exactly is the problem, stuck at StateIdle?

**Comment by ChainlessMind** [comment]
So I started a new game with one barracks to make it as easy as possible and see if I could get 8.1 to work. I select the baracks, it then tries to select the barracks (and does so at least by seeing them selected on my screen when it attempts to select them), then returns that my barracks could not be found. 

I tried everything I could find in the forum where I learned of the bot but nothing worked for me. I then added a 2nd barracks just to test it. If I select this barracks then it gives me the StateIdle message. Thanks for any help... looks like an awesome bot.
